### PR TITLE
Output PURE calls

### LINKED ISSUE #6: Annotate generated `tw` calls with PURE comment
https://babeljs.io/docs/en/babel-helper-annotate-as-pure.html

Don't see why we shouldn't do this, could bring some runtime perf savings in some cases

### PR DESCRIPTION
**Pull request by itsMapleLeaf**
Closes #6

### CODE REVIEW

**Review by sastan — APPROVED** [review]
LGTM

File: test/fixtures/tw-import/output.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 import { tw } from 'twind'
 const buttonClass = tw`bg-blue-500`
```
Comment (itsMapleLeaf) [inline test/fixtures/tw-import/output.js]: Actually, after a second look, this call should be annotated too, shouldn't it?

Comment (sastan) [reply test/fixtures/tw-import/output.js]: I didn't look at all code only at the changes. Sorry

File: src/macro.ts [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import * as babel from '@babel/core'
+import annotateAsPure from '@babel/helper-annotate-as-pure'
 import { createMacro, MacroParams } from 'babel-plugin-macros'
 import { findJsxAttributeByName, getJsxAttributeName, getJsxAttributeValue } from './jsx-attribute'
 
-function twindMacro({ state }: MacroParams) {
+const localImportName = '_macro_tw'
```
Comment (sastan) [inline src/macro.ts]: I think babel as a way to generate a local name that does not interfere with any existing name. It's a while ago ... I can not remember the function.

**Comment by itsMapleLeaf** [comment]
Changes here are done. If it looks good, we can merge and release

File: src/macro.ts (hunk 2 above)
Comment (itsMapleLeaf) [reply src/macro.ts]: Is it this? https://babeljs.io/docs/en/babel-helper-module-imports

**Comment by sastan** [comment]
I have added you to @twind/macro publishers on npm. You should be able to publish it using `yarn publish`. This script uses [np](https://www.npmjs.com/package/np) which really helps with publishing packages.

File: src/macro.ts (hunk 2 above)
Comment (sastan) [reply src/macro.ts]: Looks like it. 👍

Comment (itsMapleLeaf) [reply src/macro.ts]: Ah, I did a little digging (a.k.a stealing more ideas from twin.macro 😆) and it looks like this is a part of babel's core API: `program.scope.generateUid('tw')`

**Review by sastan — APPROVED** [review]

**Comment by sastan** [comment]
LGTM

**Pull request merged**